celery/app/defaults.py

```diff
@@ -294,7 +294,7 @@ def __repr__(self):
         cancel_long_running_tasks_on_connection_loss=Option(
             False, type='bool'
         ),
-        concurrency=Option(0, type='int'),
+        concurrency=Option(None, type='int'),
```

Comment:
can you also update the related tests?

Reply:
Yes! :grin:

Reply:
Hi @auvipy, I didn't find any related test (can you validate if I'm right?), also coverage has not changed.

Reply:
will have a closer look in the morning tomorrow. good night!

Reply:
The CLI currently has no unit tests as we didn't have the time to write them.